### ISSUE TITLE
doubt in question 1 worksheet 1

### ISSUE THREAD

**Issue by dbc2201** [dbc2201/gla-bcsc0001-2018#8]
Q : What will be the output of this code?
```c
#include <stdio.h>
int main()
{
    int a = -1;
    if (a)  //  if there is a semicolon after the if
        printf("hello");  //  and if there is an else after the printf() statement
    printf("world");
    return 0;
}
```
Q : What will be the output of this code?
```c
#include <stdio.h>
int main()
{
    int a = -1;
    if (a);
        printf("hello");
    else
        printf("world");
    return 0;
}
```
Questions :
1. What will be output of both the codes?
2. What woulf happen to the output of the second code if the value of a is 0.

**Comment by nik-1207** [comment]
there will be error in the code saying something like "using else without previous if

**Comment by dbc2201** [comment]
I have resolved this error in #7 .
